lntest/itest/lnd_test.go

```diff
@@ -985,6 +991,8 @@ func (c commitType) Args() []string {
 		return []string{"--protocol.committweak"}
 	case commitTypeTweakless:
 		return []string{}
+	case commitTypeAnchors:
+		return []string{"--protocol.anchors"}
```

Comment:
With this PR we could also change the comment for the `--protocol.anchors` flag that currently states "Won't work with watchtowers or static channel backups".

Reply:
Fixed!